src/SFA.DAS.Assessors.Api/Controllers/TrainingCoursesController.cs

```diff
@@ -62,5 +62,27 @@ public async Task<IActionResult> GetByStandardUId(string standardUId)
                 return BadRequest();
             }
         }
+
+        [HttpGet]
+        [Route("active")]
```

Comment:
So we have a get list to get everything, and now a /active to get just the active standards. Know that EPAO will want other ones like In Development, 

Are we happy that EPAO will essentially "subtract" between the two responses? And there isn't a future scenario where we want an additional filter like "in development"?

Essentially i'm debating whether this endpoint should be combined with the Get All with a filter, or left separate...

Reply:
Valid point, I debated that myself. But because I don't know what the `in-development` requirement is, I would just go with this for now and when we have good idea about in-development, we can always refactor. What I didn't wanted to do is make assessor service aware of the filters, wanted to keep that with outer api.